package.json

```diff
@@ -38,6 +38,7 @@
   },
   "devDependencies": {
     "chai": "^4.2.0",
+    "chai-http": "^4.2.0",
```

Comment:
Always increment `version` when you change the code. And do not forget to increment `minor` portion when you introduce some breaking changes (like in this pull-request). https://semver.org/